[AOTI] Support MKLDNN qlinear ops in cpp wrapper

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #134974
* #135086
* #134857
* #134795
* __->__ #134783
* #134475

Summary: Similar to https://github.com/pytorch/pytorch/pull/134475, support qlinear in the ABI-compatible mode for cpp-wrapper Inductor.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @chauhang

desertfire committed:
[AOTI] Support MKLDNN linear and qlinear ops

Summary: Similar to https://github.com/pytorch/pytorch/pull/134475, support linear and qlinear in the ABI-compatible mode for cpp-wrapper Inductor.

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Support MKLDNN linear and qlinear ops"


Summary: Similar to https://github.com/pytorch/pytorch/pull/134475, support linear and qlinear in the ABI-compatible mode for cpp-wrapper Inductor.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

huydhn commented:
@desertfire I want to test our check label script quickly using your PR, so I will remove the `topic: ` label in a min, and will put it back

desertfire committed:
Update on "[AOTI] Support MKLDNN qlinear ops in cpp wrapper"


Summary: Similar to https://github.com/pytorch/pytorch/pull/134475, support qlinear in the ABI-compatible mode for cpp-wrapper Inductor.

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames chauhang

[ghstack-poisoned]

desertfire committed:
Update on "[AOTI] Support MKLDNN qlinear ops"


Summary: Similar to https://github.com/pytorch/pytorch/pull/134475, support qlinear in the ABI-compatible mode for cpp-wrapper Inductor.

[ghstack-poisoned]

desertfire committed:
Update

[ghstack-poisoned]
